thoth/user_api/api_v1.py

```diff
@@ -392,7 +406,10 @@ def post_advise_python(
     )
 
     if not force:
+
         try:
+            _set_metric_cache_hit_advise_python(source_type=source_type.upper() if source_type else "None", cached=True)
```

Comment:
I think we want to keep the source type, thamos will provide it (as discussed in https://github.com/thoth-station/thoth-application/issues/1112) The behavior should be done based on "privileged" vs "unprivileged" workflow as discussed on the meeting.

Reply:
isn't it implemented in the metod?
```
# custom metric to expose cache hit rates for advise python endpoint
def _set_metric_cache_hit_advise_python(source_type: str, cached: bool = False):
    service_cache_hit_rate = metrics.counter(
        "thoth_cache_hit_rate",
        "Thoth cache hit rate",
        is_privileged="True" if source_type == "KEBECHET" else "False",
        is_cached="True" if cached else "False",
        service="advise_python",
        env=Configuration.THOTH_DEPLOYMENT_NAME,
    )
    return service_cache_hit_rate.inc()
```

Reply:
https://github.com/thoth-station/user-api/pull/1336 should go in first (with adjustments) - that will state whether a token was supplied (privileged) or not (unprivileged workflow without syncing logic - see https://github.com/thoth-station/thoth-application/issues/945). Once these pre-requisites will be merged, then we can differentiate which cache is hit based on input on endpoints.

Reply:
I will wait for that feature then :) Thanks @fridex

cc @KPostOffice this PR depends on: https://github.com/thoth-station/user-api/pull/1336 :D 🍻 Thanks!

Reply:
Updated in https://github.com/thoth-station/user-api/pull/1350/commits/e3641064f1c7a82970810911e72e7ed0d2c2fbd7